compare_compilers.sh

```diff
@@ -87,7 +92,7 @@ echo "Compiling the self-hosted compiler..."
 if [[ "$OS" =~ Windows ]]; then
     mingw32-make self_hosted_compiler.exe
 else
-    make self_hosted_compiler
+    $(which gmake || which make) self_hosted_compiler
```

Comment:
Maybe this would be more readable: (can't leave a github suggestion here)

```sh
elif which gmake >/dev/null; then
    # NetBSD
    gmake self_hosted_compiler
else
    make self_hosted_compiler
```

Also applies to other scripts.